packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -254,34 +255,18 @@ export const GridRootStyles = styled('div', {
       alignItems: 'center',
       gridGap: theme.spacing(1),
     },
-    [`& .${gridClasses['cell--withRenderer']}`]: {
```

Comment:
Could you update https://github.com/mui-org/material-ui-x/blob/8013d30a182139b6c1d87b23ff664f22cbf9fba3/packages/grid/_modules_/grid/gridClasses.ts#L318 and mark this class name with `// TODO v6 - remove`?

Reply:
added